corehq/messaging/scheduling/forms.py

```diff
@@ -1344,6 +1345,18 @@ class ScheduleForm(Form):
         required=False,
     )
 
+    use_user_case_for_filter_choices = [("false", "User Properties"), ("true", "User Case Properties")]
+
+    use_user_case_for_filter = ChoiceField(
+        label=gettext_lazy("Filter on"),
+        required=False,
+        initial='ALL',
```

Comment:
What's the initial 'ALL' doing if it's not one of the choices?

Reply:
Copy paste oversight